.eslintrc.yaml

```diff
@@ -35,6 +35,8 @@ rules:
   no-multiple-empty-lines:
     - 2
     - { max: 1, maxEOF: 1 }
+  no-prototype-builtins:
```

Comment:
Intéressant, pourquoi désactiver cette règle ?

Reply:
Ok, j'ai compris :
```
$ npm run lint

> pix-api@2.89.0 lint /Users/sebastien.roccaserra/Developer/Pix/pix/api
> eslint lib tests


/Users/sebastien.roccaserra/Developer/Pix/pix/api/lib/domain/services/user-service.js
  106:3  error  Possible race condition: `certificationProfile.userCompetences` might be reassigned based on an outdated value of `certificationProfile.userCompetences`                              require-atomic-updates
  108:3  error  Possible race condition: `certificationProfile.challengeIdsCorrectlyAnswered` might be reassigned based on an outdated value of `certificationProfile.challengeIdsCorrectlyAnswered`  require-atomic-updates
  135:3  error  Possible race condition: `certificationProfile.userCompetences` might be reassigned based on an outdated value of `certificationProfile.userCompetences`                              require-atomic-updates
  140:3  error  Possible race condition: `certificationProfile.challengeIdsCorrectlyAnswered` might be reassigned based on an outdated value of `certificationProfile.challengeIdsCorrectlyAnswered`  require-atomic-updates

/Users/sebastien.roccaserra/Developer/Pix/pix/api/lib/domain/usecases/create-session.js
  32:3  error  Possible race condition: `session.accessCode` might be reassigned based on an outdated value of `session.accessCode`  require-atomic-updates

/Users/sebastien.roccaserra/Developer/Pix/pix/api/tests/unit/domain/services/campaigns/campaign-code-generator_test.js
  76:45  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  87:46  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins

✖ 7 problems (7 errors, 0 warnings)
```

Reply:
Ok, j'ai essayé de corriger les nouveaux pbs de linter suite à la mise à jour de eslint ici :
- https://github.com/1024pix/pix/pull/865/commits/aa4b1ab

Les tests end to end ne sont pas passés en CI, mais on dirait des pbs de timeout